src/components/PatientProfilePage/PatientTransportDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  DialogContent,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+} from '@material-ui/core';
+
+const PatientTransportDialog = ({
```

Comment:
same thing here, can potentially be replaced by `ConfirmModal`

Reply:
The content uses <ul> and <li> tags and the button text stacks when I use ConfirmModal so I'm gonna leave this one

Reply:
we recently merged in a change to `ConfirmModal` that allows you to pass an element, not just a string - would that work for this usecase?
https://github.com/uwblueprint/paramedics-react/blob/master/src/components/common/ConfirmModal.tsx#L57